renovate.json

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "statusCheckVerify": true,
+  "ignoreDeps": [
```

Comment:
Package rules can also be used to ignore packages too, eg using patterns instead of listing everything manually. Any reason you’re permanently ignoring Babel updates and jquery though? In Renovate you can close any major update PR to mute it until one day when you eventually update manually, after which it resumes minor and patch updates. Also, you will still get minor and patch updates for your existing major version while muting the new major

Reply:
@rarkins we ignored babel because it's getting updates to the beta fairly frequently, but we're sticking with that particular rev for a bit due to breakage upstream. jQuery just because that's what we're on... This was driven by greenkeeper, which had less flexible rules, so we may revisit it.